raphtory-graphql/src/data.rs

```diff
@@ -85,94 +101,1000 @@ impl Data {
                 loader(path)
             })
     }
+}
 
-    pub fn load_from_file(path: &str) -> HashMap<String, IndexedGraph<MaterializedGraph>> {
-        fn get_graph_name(path: &Path, graph: &MaterializedGraph) -> String {
-            graph
-                .properties()
-                .get("name")
-                .into_str()
-                .map(|v| v.to_string())
-                .unwrap_or_else(|| path.file_name().unwrap().to_str().unwrap().to_owned())
-        }
-
-        fn is_disk_graph_dir(path: &Path) -> bool {
-            // Check if the directory contains files specific to disk_graph graphs
-            let files = fs::read_dir(path).unwrap();
-            let mut has_disk_graph_files = false;
-            for file in files {
-                let file_name = file.unwrap().file_name().into_string().unwrap();
-                if file_name.ends_with(".ipc") {
-                    has_disk_graph_files = true;
-                    break;
-                }
+fn copy_dir_recursive(source_dir: &Path, target_dir: &Path) -> Result<()> {
```

Comment:
this seems to be used only for `#[cfg(feature = "storage")]` can you mark it as such?